services/file-retriever/.env.sample

```diff
@@ -0,0 +1,3 @@
+API_SECRET=secret
+CORS_ORIGIN=*
+PORT=3000
```

Comment:
This would collide with the indexer port.

Reply:
True, I'll use another

Reply:
Updated to `8090`